html-css-onboarding/login.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/dist/output.css" rel="stylesheet" />
+    <title>Login</title>
+  </head>
+  <body>
+    <div class="gradient"></div>
+    <div class="background">
+      <nav class="nav-bar">
+        <div class="logo">
+          <div class="oval"></div>
+          <a href="#" class="nav-item movy">Movy</a>
+        </div>
+        <!-- <a href="#" class="nav-item mr-10">Register</a> -->
+        <a href="#" class="nav-item text-sm">Register</a>
+        <button
+          type="button"
+          class="px-5 py-0.5 ml-4 bg-blue-600 text-white text-sm leading-tight rounded shadow-md hover:bg-blue-700 hover:shadow-lg focus:bg-blue-700 focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out border-none"
+        >
+          Login
+        </button>
+      </nav>
+      <div class="block p-6 rounded-lg shadow-lg bg-white max-w-sm login-form">
+        <h2 class="font-medium leading-tight text-4xl mt-0 text-center">
+          Inicia sesión
+        </h2>
+        <form>
+          <div class="form-group mb-5">
+            <input
+              type="email"
+              class="form-control px-3 py-1.5 text-base font-normal bg-white bg-clip-padding border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
+              id="exampleInputEmail2"
+              aria-describedby="emailHelp"
+              placeholder="Email o número de teléfono"
+            />
+          </div>
+          <div class="form-group mb-11">
+            <input
+              type="password"
+              class="form-control block px-3 py-1.5 text-base font-normal bg-white bg-clip-padding border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
+              id="exampleInputPassword2"
+              placeholder="Contraseña"
+            />
+          </div>
+          <button
+            type="submit"
+            class="w-full px-6 py-2.5 mb-6 bg-blue-600 text-white text-base font-normal leading-tight rounded shadow-md hover:bg-blue-700 hover:shadow-lg focus:bg-blue-700 focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out border-none"
+          >
+            Iniciar sesión
+          </button>
+          <div
+            class="flex justify-between items-center mb-7 flex-col md:flex-row"
+          >
+            <div class="form-group form-check mb-2 md:mb-0">
+              <input
+                type="checkbox"
```

Comment:
This checkbox doesn't look perfectly aligned with it's label and other content in the mid section of the form, is there something we can do to align this?